packages/client/src/v2-events/features/events/registered-fields/AlphaPrintButton.tsx

```diff
@@ -62,7 +68,7 @@ export const AlphaPrintButton = {
     const intl = useIntl()
     const location = useLocation()
     const parts = location.pathname.split('/')
-    const eventId = UUID.parse(parts[3])
+    const eventId = parts[3] as UUID
```

Comment:
Let's add a comment here mentioning why we had to assert the type. I'm guessing it's due to the fact that the id could be the temporary one